Assets/Art Assets/Downloaded Asset Packs.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: ddd16074c7f4ec24a916d016e9b4e7e2
+guid: a05bfa56cc654e547b7d36a61f9e6cfd
```

Comment:
was this auto generated? or is there a significance to this id string?

Reply:
Hmm yeah I'm not sure what caused it but was auto generated I think?
I created the Downloaded Asset Packs folder so I'm not sure how I could've modified it in one commit. 🤷🏻
Tried creating and renaming folders but wasn't able to replicate so I'm not sure unfortunately I'll keep an eye out for that in the future.